chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -594,6 +594,8 @@ zotero.preferences.advanced.debug.error = An error occurred sending debug output
 
 dragAndDrop.existingFiles = The following files already existed in the destination directory and were not copied:
 dragAndDrop.filesNotFound = The following files were not found and could not be copied:
+dragAndDrop.invalidLink.title = File Not Found
+dragAndDrop.invalidLink.text = The following file shortcut points to a non-existent file and cannot be imported: \n\n%S
```

Comment:
For newlines we can just add in the code rather than having them in the string (where they're prone to getting mangled).